doc/api/process.md

```diff
@@ -1103,6 +1103,25 @@ and [Cluster][] documentation), the `process.connected` property will return
 Once `process.connected` is `false`, it is no longer possible to send messages
 over the IPC channel using `process.send()`.
 
+## `process.constrainedMemory()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+Gets the amount of memory available to the process (in bytes) based on
+limits imposed by the OS. If there is no such constraint, or the constraint
+is unknown, `0` is returned.
+
+It is not unusual for this value to be less than or greater than `os.totalmem()`.
+This function currently only returns a non-zero value on Linux, based on cgroups
```

Comment:
This sentence is worded a bit confusingly. What is the value on Windows?

Reply:
The description was copied from the [Libuv](https://docs.libuv.org/en/v1.x/misc.html#c.uv_get_constrained_memory) documentation.

Reply:
That may be true but it's still confusing as is. Not sure I have a better suggestion tho